src/common/common-util/src/main/java/ee/ria/xroad/common/Version.java

```diff
@@ -92,24 +96,22 @@ public final class Version {
 
     /**
      * Outputs version information and a warning if JVM version is not in the given range
-     * @param minJavaVersion the minimum supported version
-     * @param maxJavaVersion the maximum supported version
      */
-    public static void outputVersionInfo(String appName, int minJavaVersion, int maxJavaVersion) {
+    public static void outputVersionInfo(String appName) {
         // print app name + version and java vendor name + runtime version
         int javaVersion = readJavaVersion();
         String runtimeName = System.getProperty(JAVA_RUNTIME_NAME_PROPERTY, JAVA_DEFAULT_RUNTIME_NAME);
         String vendorVersion =  JAVA_VENDOR != null ? javaVersion + " "  + JAVA_RUNTIME_VERSION : JAVA_RUNTIME_VERSION;
 
         log.info(String.format("%s %s (%s %s)", appName, XROAD_VERSION, runtimeName, vendorVersion));
-        if (javaVersion < minJavaVersion || javaVersion > maxJavaVersion) {
-            if (minJavaVersion == maxJavaVersion) {
+        if (javaVersion < MIN_SUPPORTED_JAVA_VERSION || javaVersion > MAX_SUPPORTED_JAVA_VERSION) {
+            if (MIN_SUPPORTED_JAVA_VERSION == MAX_SUPPORTED_JAVA_VERSION) {
                 log.warn("Warning! Running on unsupported Java version {}. Java version {} is currently supported.",
-                        javaVersion, minJavaVersion);
+                        javaVersion, MIN_SUPPORTED_JAVA_VERSION);
             } else {
                 log.warn(
                         "Warning! Unsupported Java version {}. Java versions {} - {} are currently supported.",
-                        javaVersion, minJavaVersion, maxJavaVersion);
+                        javaVersion, MIN_SUPPORTED_JAVA_VERSION, MAX_SUPPORTED_JAVA_VERSION);
```

Comment:
If MIN and MAX are both equal to 11, then here we have a case where it always will say that it's unsupported. Don't we need => or =< ?

Reply:
```java
if (javaVersion < MIN_SUPPORTED_JAVA_VERSION || javaVersion > MAX_SUPPORTED_JAVA_VERSION)
```
Given that MIN & MAX both are 11 and if runtime version is 11 it will skip this block & no warning will be logged